sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -346,7 +348,9 @@ public enum UserRight {
 	public static final String _CAMPAIGN_FORM_DATA_EXPORT = "CAMPAIGN_FORM_DATA_EXPORT";
 	public static final String _BAG_EXPORT = "BAG_EXPORT";
 	public static final String _SORMAS_TO_SORMAS_SHARE = "SORMAS_TO_SORMAS_SHARE";
-	public static final String _LAB_MESSAGES = "LAB_MESSAGES";
+	public static final String _EXTERNAL_MESSAGE_VIEW = "EXTERNAL_MESSAGE_VIEW";
+	public static final String _EXTERNAL_MESSAGE_PROCESS = "EXTERNAL_MESSAGE_PROCESS";
```

Comment:
There is a [cronjob for fetching external messages](https://github.com/hzi-braunschweig/SORMAS-Project/blob/1b4d879cb1919b941757732451dd01b660976130/sormas-backend/src/main/java/de/symeda/sormas/backend/common/CronService.java#L202). It should still work after the user rights refactoring, or?
By the way, the method in the CronService is still called `fetchLabMessages()`, which is not appropriate anymore.